scripts/apps/search/views/save-search-dialog.html

```diff
@@ -13,5 +13,9 @@
             <span id="search_global" sd-switch ng-model="edit.is_global"></span>
             <label class="label--inline" translate>Make global</label>
         </div>
+        <div class="field" ng-if="edit.is_global">
```

Comment:
Doesn't this need the same feature flag check as in https://github.com/superdesk/superdesk-client-core/pull/2925/files#diff-275cb0eb9ee87557fa6c69e095e061deR85 ?